### PR TITLE
Fix VDSFC bit mismatch for vaapi

### PR DESCRIPTION
**Pull request by shawnli2**
value of chroma sample location was not set in lib

Issue: MDP-66020
Test: manually

### CODE REVIEW

File: _studio/shared/umc/codec/h264_dec/src/umc_h264_va_packer.cpp [hunk 1]
```diff
@@ -731,6 +731,7 @@ void PackerVA::PackProcessingInfo(H264DecoderFrameInfo * sliceInfo)
 
     pipelineBuf->surface = m_va->GetSurfaceID(sliceInfo->m_pFrame->m_index); // should filled in packer
     pipelineBuf->additional_outputs = (VASurfaceID*)vpVA->GetCurrentOutputSurface();
+    pipelineBuf->input_color_properties.chroma_sample_location = VA_CHROMA_SITING_HORIZONTAL_LEFT | VA_CHROMA_SITING_VERTICAL_CENTER;
```
Comment (dmitryermilov) [inline _studio/shared/umc/codec/h264_dec/src/umc_h264_va_packer.cpp:734]: Please add some comment **in code** why exactly "HORIZONTAL_LEFT |  VERTICAL_CENTER" is set. It's not obvious why this hardcoded value is chosen.

File: _studio/shared/umc/codec/h265_dec/src/umc_h265_va_packer_vaapi.cpp [hunk 2]
```diff
@@ -113,7 +113,7 @@ namespace UMC_HEVC_DECODER
         MFX_INTERNAL_CPY(pipelineBuf, &vpVA->m_pipelineParams, sizeof(VAProcPipelineParameterBuffer));
 
         pipelineBuf->surface = m_va->GetSurfaceID(sliceInfo->m_pFrame->m_index); // should filled in packer
-        pipelineBuf->additional_outputs = (VASurfaceID*)vpVA->GetCurrentOutputSurface();
+        pipelineBuf->input_color_properties.chroma_sample_location = VA_CHROMA_SITING_HORIZONTAL_LEFT | VA_CHROMA_SITING_VERTICAL_CENTER;
```
Comment (dmitryermilov) [inline _studio/shared/umc/codec/h265_dec/src/umc_h265_va_packer_vaapi.cpp:116]: same

**Review by dmitryermilov — CHANGES_REQUESTED** [review]

File: _studio/shared/umc/codec/h264_dec/src/umc_h264_va_packer.cpp (hunk 1 above)
Comment (shawnli2) [reply _studio/shared/umc/codec/h264_dec/src/umc_h264_va_packer.cpp:735]: > Please add some comment **in code** why exactly "HORIZONTAL_LEFT | VERTICAL_CENTER" is set. It's not obvious why this hardcoded value is chosen.

Upadate code comment:     // To keep output aligned, decode downsampling use this fixed combination of chroma sitting type.

File: _studio/shared/umc/codec/h265_dec/src/umc_h265_va_packer_vaapi.cpp (hunk 2 above)
Comment (shawnli2) [reply _studio/shared/umc/codec/h265_dec/src/umc_h265_va_packer_vaapi.cpp:117]: Upadate code comment: // To keep output aligned, decode downsampling use this fixed combination of chroma sitting type

**Review by dmitryermilov — APPROVED** [review]

**Pull request merged**